### DISCUSSION TITLE
Can't see any repositories

### DISCUSSION THREAD

**Discussion by kw21106** [kubeapps/kubeapps#2322] [Q&A] [answered]
I cannot see any app in Catalog section, when I tried to manage the repositories and repo, it says Validation Failed. Got: Code: 400. Message: Get https://charts.bitnami.com/bitnami/index.yaml: x509: certificate is valid for testexp, not charts.bitnami.com

How to solve it? thanks

**Comment by andresmgot** [comment, marked as answer]
Hi @kw21106,

If your cluster is behind a proxy, you will need to setup environment variables (http_proxy, https_proxy) or even a custom CA certificate. In this thread you can find more information about it: https://github.com/kubeapps/kubeapps/issues/2139

Let us know if that helped you fix your issue.
